src/platform/3ds/3ds_ui.cpp

```diff
@@ -220,6 +227,13 @@ void CtrUi::ProcessEvents() {
 		touch_y = pos.py;
 
 		keys[keys_tbl[col + (row * 4)]] = true;
+
+		if (!touchscreen) {
+			touchscreen = true;
+			gspLcdInit();
```

Comment:
Please with error checking:
```suggestion
			if (R_SUCCEEDED(gspLcdInit())) {
				GSPLCD_PowerOnBacklight(GSPLCD_SCREEN_BOTTOM);
				gspLcdExit();
			}
```